paper-dropdown-menu-shared-styles.js

```diff
@@ -39,6 +39,12 @@ $_documentContainer.innerHTML =
         @apply --paper-dropdown-menu;
       }
 
+      :host(:dir(rtl)) {
+        text-align: right;
+
+        @apply(--paper-dropdown-menu);
```

Comment:
The `:host` ruleset above already applies this mixin, is there a reason to apply it again?

Reply:
Reason is if they manually had already overrode `text-align`, applying this mixin here is a backcompat. I did not do this in paper-dropdown-menu-lite because they already had rules that did not take this into account, so that would be a breaking change on that front

Reply:
I think we might have discussed this in person yesterday but could the `text-align: left;` be removed from `:host` instead?